test/object-store/primitive_list.cpp

```diff
@@ -199,20 +200,20 @@ struct String : Base<PropertyType::String, StringData> {
 
 struct Binary : Base<PropertyType::Data, BinaryData> {
     using Boxed = std::string;
-    static std::vector<BinaryData> values()
-    {
-        return {BinaryData("a", 1)};
-    }
     static util::Any to_any(BinaryData value)
     {
         return value ? std::string(value) : util::Any();
     }
+    static std::vector<BinaryData> values()
+    {
+        return {BinaryData("c", 1), BinaryData("a", 1), BinaryData("b", 1)};
+    }
 };
 
 struct Date : Base<PropertyType::Date, Timestamp> {
     static std::vector<Timestamp> values()
     {
-        return {Timestamp(1, 1)};
+        return {Timestamp(3, 3), Timestamp(1, 1), Timestamp(2, 2)};
```

Comment:
Unsure if this is a merge or if you added this, but given the new values here, Date::max() needs to return `Timestamp(3, 3)` now.